config/webpack.config.js

```diff
@@ -29,7 +29,10 @@ module.exports = {
                 test: /\.ts$/,
                 loader: 'ts-loader',
                 options: {
-                    configFile: path.resolve('./config/tsconfig.json')
+                    configFile: path.resolve('./config/tsconfig.json'),
+                    compilerOptions: {
+                        target: 'es5'
```

Comment:
Not sure why you changed the tsconfig.json to use ES6, only to force es5 here ?

Reply:
The es6 build will build es6 instead of es5